src/test/rpc_wallet_tests.cpp

```diff
@@ -383,12 +383,13 @@ BOOST_AUTO_TEST_CASE(rpc_wallet_z_validateaddress)
     BOOST_CHECK_NO_THROW(retValue = CallRPC("z_validateaddress zs1z7rejlpsa98s2rrrfkwmaxu53e4ue0ulcrw0h4x5g8jl04tak0d3mm47vdtahatqrlkngh9slya"));
     resultObj = retValue.get_obj();
     b = find_value(resultObj, "isvalid").get_bool();
-    BOOST_CHECK_EQUAL(b, true);
-    BOOST_CHECK_EQUAL(find_value(resultObj, "type").get_str(), "sapling");
-    b = find_value(resultObj, "ismine").get_bool();
+    // TODO: Revert when we re-enable Sapling addresses on mainnet
```

Comment:
Not clear what gets reverted and developer will have to check commit.  How about keeping old test suitably commented between /** and */ with instructions? Maybe use in conjunction with #if 1 , #if 0 blocks.

Reply:
The way this PR is designed, all we need to do is a `git revert COMMIT_ID` for the last commit in this PR (that disabled Sapling support), and everything will be correctly reverted.